doc/generic/pgf/text-en/pgfmanual-en-tutorial.tex

```diff
@@ -1046,7 +1046,7 @@ \subsection{Repeating Things: For-Loops}
 them, feel free to use them. \tikzname\ introduces yet another command, called
 |\foreach|, which I introduced since I could never remember the syntax of the
 other packages. |\foreach| is defined in the package |pgffor| and can be used
-independently \tikzname, but \tikzname\ includes it automatically.
+independently by \tikzname, but \tikzname\ includes it automatically.
```

Comment:
From [this question on English SE](https://english.stackexchange.com/q/39349) I think _of_ would be better: `can be used independently of \tikzname`.